### DISCUSSION TITLE
2024年8月6日已经缴费续期2年、1个月多了、邮箱登录、授权设备使用专业版时收不到验证码。邮件、git论坛反应后一直都没回应！

### DISCUSSION THREAD

**Discussion by strayfood** [getlantern/lantern-forum-cn#877] [Q&A]
2024年8月6日已经缴费续期2年、1个月多了直到现在还是邮箱登录、授权设备使用专业版时收不到验证码。
@Mike777Khoo 
邮件、git论坛反应后一直都没回应、没回复！

支付记录截图：（信用卡尾号是9335，2024-08-06支付的619元，邮箱填写的就是现在的这个: jobsbildad@gmail.com
![E0BB49D1-7EB9-429A-B8B9-4A1F6527FF31_4_5005_c](https://github.com/user-attachments/assets/364c85b0-51b2-43bf-a9a2-eba80bf926fe)

**Comment by MaxMKMax** [comment]
@strayfood 您好. 我没权限处理邮箱的. 非常抱歉让您久等了. 马上给您提交问题上去处理.

**Comment by MaxMKMax** [comment]
@strayfood 非常抱歉!

**Comment by greySkiesMarsh** [comment]
Hello @strayfood 

I wasn't able to track down your account due to what seems like a database error on our end. 

Please use the following code to activate with the email you posted in this thread, and I can update your plan expiration in accordance with your purchase.

MJ343-R3GXY-D6PQY-4WX6T-K9MXG

**Reply by strayfood** [reply]
激活了。
显示专业版到期日：10/24/2024
请更新购买计划。

**Reply by MaxMKMax** [reply]
@strayfood 好的. 帮您提醒支付团队了.

**Reply by greySkiesMarsh** [reply]
@strayfood Apologies for the delay. Your plan has been updated.
